src/sentry/dynamic_sampling/utils.py

```diff
@@ -43,3 +54,54 @@ def generate_uniform_rule(project: Project) -> UniformRule:
         },
         "id": UNIFORM_RULE_RESERVED_ID,
     }
+
+
+class DynamicSamplingFeatureMultiplexer:
```

Comment:
wdyt to create new file ./sentry/dynamic_sampling/feature_multiplexer.py and move this class there?

Reply:
agreed